pkg/galaxy/server.go

```diff
@@ -218,6 +221,32 @@ func (g *Galaxy) resolveNetworks(req *galaxyapi.PodRequest, pod *corev1.Pod) ([]
 	return networkInfos, nil
 }
 
+func (g *Galaxy) getNetworkConf(networkName string) map[string]interface{} {
+	if netConf, ok := g.netConf[networkName]; ok {
+		return netConf
+	}
+	// In the absence of existing network config from json
+	// config, load and execute a CNI .configlist
+	// or .config (in that order) file on-disk whose JSON
+	// “name” key matches this Network object’s name.
+	data, err := cniutil.GetNetworkConfig(networkName, g.NetworkConfDir)
+	if err != nil {
+		glog.Warningf("can't load network config %s from confdir %s", networkName, g.NetworkConfDir)
+		return nil
+	}
+	var m map[string]interface{}
+	if err := json.Unmarshal(data, &m); err != nil {
+		glog.Warningf("failed to unmarshal networkinfo %s: %v", string(data), err)
+		return nil
+	}
+	// kubeconfig of host filesystem won't be reachable for galaxy.
+	// Since galaxy is running in pod, it can talk to apiserver via secret token
+	if m["kubeconfig"] != "" {
```

Comment:
kubeconfig is useful anywhere else? if not, we should assume where's no kubeconfig entry, or when init k8sClient, kubeconfig entry should not be readed, so the if statement could be removed

Reply:
Galaxy doesn't depend on `kubeconfig`, you can check [galaxy-etc configmap](https://github.com/tkestack/galaxy/blob/master/yaml/galaxy.yaml#L149-L178).
This is to ensure other cnis which depend on kubeconfig to talk to apiserver to be continue working via service token within galaxy pod.